lib/src/base-styles/_base.scss

```diff
@@ -9,6 +9,8 @@
  * any consumer of the react-shared-components lib
  */
 
+@use '~@fortawesome/fontawesome-svg-core/styles';
```

Comment:
Important, open question: how comfortable are we that this syntax is supported in all supported downstream build configurations? It should work anywhere a typical webpack sass-loader setup is being used, as far as I know.